Dockerfile

```diff
@@ -81,5 +81,6 @@ RUN R -e 'renv::install("pacman")'
 WORKDIR /workspace
 
 RUN R -e 'renv::snapshot(type = "all")'
+RUN R -e 'write.csv(installed.packages()[, c("Package","Version")], row.names=FALSE, file="/packages.csv")'
```

Comment:
Isn't this step now redundant?

Reply:
I thought it didn't hurt to have the list in the docker image anyway.